src/js/dataEntryPanel.js

```diff
@@ -164,4 +161,20 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         floe.chartAuthoring.percentage.render(that.locate("totalPercentage"), percentage, that.options.strings.totalPercentage);
     };
 
+    floe.chartAuthoring.dataEntryPanel.resetPanel = function (that) {
```

Comment:
Good idea - something like an "updateDataEntryPanel" function. This could be used for a number of different purposes:
- loading example data
- resetting
- importing

I'll do this as part of addressing the previous comment about addExampleData.